intelmq/tests/bots/experts/uri/test_expert.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-FileCopyrightText: 2022 gutsohnCERT
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# -*- coding: utf-8 -*-
+"""
+Testing uri.
+"""
+
+import unittest
+import intelmq.lib.test as test
+import os
+if os.getenv('INTELMQ_TEST_EXOTIC'):
+    from intelmq.bots.experts.uri.expert import URIExpertBot
```

Comment:
```suggestion
from intelmq.bots.experts.uri.expert import URIExpertBot
```

You can remove the os.getenv, because its handled by the other suggestion below, using @test.skip_exotic()